runtime/common/src/weights/pallet_sudo.rs

```diff
@@ -0,0 +1,75 @@
+//! Autogenerated weights for pallet_sudo
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-05-09, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `bm3`, CPU: `Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// target/production/substrate
+// benchmark
+// pallet
+// --steps=50
+// --repeat=20
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --json-file=/var/lib/gitlab-runner/builds/zyw4fam_/0/parity/mirrors/substrate/.git/.artifacts/bench.json
+// --pallet=pallet_sudo
+// --chain=dev
+// --header=./HEADER-APACHE2
+// --output=./frame/sudo/src/weights.rs
+// --template=./.maintain/frame-weight-template.hbs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
```

Comment:
copied this from substrate and I don't think we can currently ran the benchmarks for it. The reason is that we can not add `pallet_sudo` into list of benchmarks due to it only being in certain features (and macro didn't supporting that) and we might need to run it manually to get these value but if there Frequency reference hardware is close enough to the Polkadot then we might be able to just use these values.

Reply:
I think that the copy-paste is fine:

1. Rococo and local only
2. Sudo usually overrides the weight anyway.